src/gui/src/mainWindow.cpp

```diff
@@ -400,9 +400,11 @@ void MainWindow::init(sta::dbSta* sta)
   timing_widget_->init(sta);
   controls_->setSTA(sta);
   hierarchy_widget_->setSTA(sta);
-
-  // register descriptors
+  // register renderers
   auto* gui = Gui::get();
+  guide_renderer_ = std::make_unique<GuideRenderer>(viewer_->getRouteGuides());
+  gui->registerRenderer(guide_renderer_.get());
```

Comment:
DRT or GRT could hold on to this renderer instead of making it part of the main_window. We can figure out a way to pass the actions along to the layout viewer